timer.go

```diff
@@ -0,0 +1,26 @@
+package dbft
+
+import (
+	"time"
+
+	timer2 "github.com/nspcc-dev/dbft/timer"
+)
+
+// Timer is an interface which implements all time-related
+// functions. It can be mocked for testing.
+type Timer interface {
+	// Now returns current time.
+	Now() time.Time
+	// Reset
+	Reset(s timer2.HV, d time.Duration)
```

Comment:
I think it'd be inconvenient to use without `dbft.HeightView`

Reply:
Fixed!